etc/wordpress.json

```diff
@@ -0,0 +1,20 @@
+{
+  "consul": "consul:8500",
+  "services": [
+    {
+      "name": "wordpress",
+      "port": 80,
+      "publicIp": true,
+      "health": "/usr/bin/curl --fail -s http://localhost/health",
```

Comment:
Implemented, but untested in https://github.com/misterbisson/triton-wordpress/pull/1/files#diff-fb6924f36518eed234bc93682c25dec1